mailers/password_mailer/password_mailer.go

```diff
@@ -1,9 +1,81 @@
 package password_mailer
 
-// TODO: ドメイン取得後にSESによる送信を行う
-// https://github.com/sourcegraph/go-ses
-func Reset(email string, token string) {
+import (
+	"../../config"
+	"../../modules/logging"
+	"os"
+
+	"github.com/aws/aws-sdk-go/aws"
+	"github.com/aws/aws-sdk-go/aws/session"
+	"github.com/aws/aws-sdk-go/service/ses"
+)
+
+func Reset(id int64, address string, token string) {
```

Comment:
__[golint]__
* [golint] exported function Reset should have comment or be unexported